docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -46,9 +46,13 @@ public static void Example()
                 IDataView shuffledFullImagesDataset = mlContext.Data.ShuffleRows(
                     mlContext.Data.LoadFromEnumerable(images));
 
-                shuffledFullImagesDataset = mlContext.Transforms.Conversion
-                    .MapValueToKey("Label")
-                    .Fit(shuffledFullImagesDataset)
+                var estimator = mlContext.Transforms.Conversion
+                    .MapValueToKey("Label");
+                var estimatorWithKeyType = estimator.Append(
+                    mlContext.Transforms.Conversion.MapKeyToValue(
+                        outputColumnName: "LabelAsKey", inputColumnName: "Label"));
+
+                shuffledFullImagesDataset = estimatorWithKeyType.Fit(shuffledFullImagesDataset)
```

Comment:
There shouldn't a need to modify this part of code. You will just need to add a transform after ImageClassification that gets Key mappings from PredictedLabel column by doing MapKeyToValue.